### ISSUE TITLE
[Enhancement] Ability to update account contact information

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/reserve-rec-public#63]
#### Description:

As a park account holder, I want to be able to update my account so that my contact information is up-to-date and accurate.

[Description of the ticket]

#### Acceptance Criteria:

Given
When
Then 

Given
When
Then

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Wireframe(s):**

**Workflows:**

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by LindsayMacfarlane** [comment]
Backlog refinement, June 23: @pavelcreates flagged that designs are ready in 'ready-for dev' in Figma.

https://www.figma.com/design/1QHd1cWiRqeC0Ua0E9nfhr/PRDT---Public?node-id=6231-189468&t=Qv5Avo7Rug8nDQtF-4

@Christopher-walsh22 - flags that we will need to define what provinces/stat we want to display in drop down. Jess flags that we've had issues in the past, so can we allow people to manually enter. Dave mentions we can do Canada Post API for Canadian addresses and then text input for non-Canadians. Meredith mentions that there may be a library somewhere we can leverage for other countries.

**Comment by LindsayMacfarlane** [comment]
Backlog refinement June 30, 2026 : @Christopher-walsh22 flags we can move forward with @meredithom suggest on using a library. Chris also flags that this links to other tickets within the EPIC and that he updated order.

**Comment by manuji** [comment]
Tested on TEST: **Pending**
1. Account details page does not reflect designs and reported here: https://github.com/bcgov/reserve-rec-public/issues/634

2. Users with a park/guest account **cannot** update the email address and password. 
<img width="460" height="388" alt="Image" src="https://github.com/user-attachments/assets/22d77e1c-d355-4191-935a-686b9afec091" />

3. When you save the changes, the edit form retains on the page and displayed with duplicate sections as below;
<img width="1178" height="882" alt="Image" src="https://github.com/user-attachments/assets/90229497-5713-4f51-98d7-2d5a7b1d4f85" />

<img width="949" height="310" alt="Image" src="https://github.com/user-attachments/assets/5c4ce1af-6939-42d4-8a1b-34520ea37a67" />

**Please note** this task ticket cannot be tested with the BCSC account as the option to login with BCSC does not function. fyi @LindsayMacfarlane .

**Comment by manuji** [comment]
Tested on TEST: Pending
* I still cannot edit the Account management section items. 

<img width="1386" height="742" alt="Image" src="https://github.com/user-attachments/assets/507d3c2a-5b0b-46f9-888b-9135c7d2e72c" />

**Comment by marklise** [comment]
@manuji reproduces, but it's intentional — both buttons were shipped `disabled` in #624 and deferred to #643 (password) and #644 (email address).

The AC above still lists email and password as editable, so this will keep coming back Pending. @LindsayMacfarlane can we strike those two and let #643/#644 carry them?

Your other two items from July 16 are fixed in v0.0.77: designs under #634, duplicate sections after save in `b5f5393`.

Remaining scope to test here: contact information, vehicle details, and the BCSC lock (still blocked while BCSC login is down).

**Comment by LindsayMacfarlane** [comment]
@marklise @manuji - I support Mark's recommendation. Two new tickets (as indicated above) have been subsequently created to address.

**Comment by manuji** [comment]
Tested on TEST: **Done**
* Verified that the Acceptance criteria is met for the scope approved by @LindsayMacfarlane . 
* Enhancement ticket created for a bug: https://github.com/bcgov/reserve-rec-public/issues/664
* Passing to PO review.

**Comment by LindsayMacfarlane** [comment]
Approved. 

FYI @manuji - I noticed a potential bug when testing where the account page stayed visible even though I logged out. Could you please double check?

**Comment by manuji** [comment]
@LindsayMacfarlane it's already reported here: https://github.com/bcgov/reserve-rec-public/issues/631

**Comment by LindsayMacfarlane** [comment]
Of course it is @manuji - you are amazing!
